packages/react-native-hms/src/classes/HmsView.tsx

```diff
@@ -91,10 +91,6 @@ export const HmsViewComponent = React.forwardRef<any, HmsComponentProps>(
     const capture = async () => {
       const viewManagerConfig = UIManager.getViewManagerConfig('HMSView');
 
-      if (!viewManagerConfig.Commands.capture) {
```

Comment:
@ygit What happens when `viewManagerConfig.Commands.capture` is `undefined` or `null`?

Reply:
Undefined behaviour would happen. It should never be `undefined` or `null`.